tests/fs/test_strip_unstrip.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from dvc_objects.fs.implementations import (
+    azure,
+    gs,
+    hdfs,
+    http,
+    https,
+    oss,
+    s3,
+    ssh,
+    webhdfs,
+)
+from dvc_objects.fs.base import FileSystem
+
+
+@pytest.fixture(autouse=True)
+def mock_check_requires(mocker):
+    yield mocker.patch.object(FileSystem, "_check_requires")
```

Comment:
I am trying not to avoid dependency on filesystems as much as possible. `dvc-objects` should not really depend on real implementations, but to their interface.